environments/kubernetes/manifests/thanos-querier-cache-configmap.yaml

```diff
@@ -0,0 +1,26 @@
+apiVersion: v1
+data:
+  observatorium-cache-config.yaml: |-
+    "auth_enabled": false
+    "frontend":
+      "align_queries_with_step": true
+      "cache_results": true
+      "compress_responses": true
+      "results_cache":
+        "cache":
+          "enable_fifocache": true
```

Comment:
why not memcache? It's easy to operate

Reply:
We'd like to keep the dependencies to a minimum here, considering that this is a stop-gap measure till thanos querier natively gets features like caching+query splitting. 

I do like your suggestion of adding memcache. Perhaps this can be independently of the PR as the next iteration?

Reply:
Yes, of course, but I think it's must have overall. Feel free to do it in next PR :+1: